HSLdevcom/jore4-ui
# 35325 - Line details observation date outside its validity period box

Added box to line details to show information when observation date is outside its validity period.

Testing:

1. Go to Routes and lines (/routes)
2. Click line
3. Select line
4. Set observation date what is outside line's validity period
5. Should show info box with alert


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/HSLdevcom/jore4-ui/940)
